### ISSUE TITLE
Implement auto reresh at `/activetask/map`

### ISSUE THREAD

**Issue by bsaiken** [FWAJL/FieldWorkManager#1155]
May be done with #1154.

**Comment by bsaiken** [comment]
@damirius 
Damir - this seemed the easiest way to contact you.  I talked to Jeremie the other day and understand you are working for him.  Great news.

I have enrolled in graduate school in Maryland.  I am working on a project where my web site would be useful - even in the unfinished state that we left it.  However, the map needs to work and I get an error.  

I think we are just missing an updated api key.  I couldn't figure out how to update it.

http://stackoverflow.com/questions/2769148/whats-the-api-key-for-in-google-maps-api-v3

Is this something you would be willing to help me with?

Brian

**Comment by damirius** [comment]
@bsaiken

Yes it seems that api key is missing. Do you get "oops something went wrong" error when loading pages with the map?

Positive side is that I'm almost 100% sure that the problem lies with usage of api key, negative side is that we specified gmaps api js url inside routes.xml and we can't just set up api key in the config to solve this.
You can go to `/Applications/PMTool/Config/routes.xml` and change every line that looks like this:

`https://maps.googleapis.com/maps/api/js?libraries=drawing,geometry` to this
`https://maps.googleapis.com/maps/api/js?key=[APIKEY]&amp;libraries=drawing,geometry`
Where [APIKEY] is your api key registered for browser usage in google developer console.
